src/entities/TimeLog/index.ts

```diff
@@ -124,7 +124,12 @@ export class ExtendedTimeLog extends TimeLog {
                 const ms = reader.nextUint32()
                 const days = reader.nextUint16()
 
-                record.time = new Date(1000 * 60 * 60 * 24 * days + ms) //TODO: timezone ?
+                // Date-class constructor requires milliseconds since 1970-01-01
+                // Binary file provides days since 1980-01-01 (see ISO for reference)
+                // therefore add ten years to get the correct time
+                const TEN_YEARS_MILLISECONDS = 10 * 365 * 24 * 60 * 60 * 1000;
```

Comment:
This value is not accurate, it doesn't take into account leap years between 1970 and 1980. The exact number of milliseconds in those 10 years is `315532800000`.